app/utils/galaxy-api.ts

```diff
@@ -0,0 +1,44 @@
+import ky from "ky";
+
+interface WorkflowLandingsBody {
+  request_state: { reference_genome: string };
```

Comment:
This is going to vary across workflows, I suppose
```suggestion
  request_state: { [key: string]: string }:
```
would be the right interface ?